hw/dv/doc/dv_plan_template.md

```diff
@@ -36,12 +36,12 @@ FOO testbench has been constructed based on the [CIP testbench architecture]({{<
 ### Top level testbench
 Top level testbench is located at `hw/ip/foo/dv/tb/tb.sv`. It instantiates the FOO DUT module `hw/ip/foo/rtl/foo.sv`.
 In addition, it instantiates the following interfaces and sets their handle into `uvm_config_db`:
-* [Clock and reset interface]({{< relref "hw/dv/sv/common_ifs/README" >}})
+* [Clock and reset interface]({{< relref "hw/dv/sv/common_ifs/index" >}})
 * [TileLink host interface]({{< relref "hw/dv/sv/tl_agent/README.md" >}})
 * FOO IOs
-* Interrupts ([`pins_if`]({{< relref "hw/dv/sv/common_ifs/README.md" >}}))
-* Alerts ([`pins_if`]({{< relref "hw/dv/sv/common_ifs/README.md" >}}))
-* Devmode ([`pins_if`]({{< relref "hw/dv/sv/common_ifs/README.md" >}}))
+* Interrupts ([`pins_if`]({{< relref "hw/dv/sv/common_ifs/index.md" >}}))
```

Comment:
Drop the `.md` everywhere based on @gkelly' comment above?

Reply:
Don't link to `index.md` files either, this should be `{{< relref "hw/dv/sv/common_fs" >}}` as that's the name of the page bundle itself.

Reply:
Thanks @sriyerg and @gkelly . I have committed required updates.